src/vs/editor/browser/controller/mouseHandler.ts

```diff
@@ -424,6 +450,36 @@ class MouseDownOperation extends Disposable {
 		// Overwrite the detail of the MouseEvent, as it will be sent out in an event and contributions might rely on it.
 		e.detail = this._mouseState.count;
 
+		if (!this._mouseState.altKey // we don't support multiple mouse
```

Comment:
We should also include a test for an editor option here. `editor.dragAndDrop` or something... `this._context.configuration.editor` is available in this class.